spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/WellKnownAuthParameters.java

```diff
@@ -28,40 +28,90 @@ public final class WellKnownAuthParameters {
 
 	private static final Map<String, String> lowerCaseToCamelCase = new HashMap<>();
 
+	private static final String TENANT_DOMAIN = "tenantDomain";
```

Comment:
This is definitely an impovement @alpreu . If we made these an enum we could then also iterate through them to add their camel case mapping and avoid the N calls for each of these below. Wdyt?